chainer/utils/sparse.py

```diff
@@ -14,6 +14,14 @@ class CooMatrix(object):
         col (numpy.ndarray or cupy.ndarray): The column indices of the matrix
             entries.
         shape (tuple of int): The shape of the matrix in dense format.
+        order ('C', 'F', 'other', None): If ``'C'``, the maxtix is assumed that
```

Comment:
How about adding "or" between `'other'` and `None` to be less ambiguous and more consistent.